subprojects/core/src/main/groovy/org/gradle/api/file/ContentFilterable.java

```diff
@@ -61,7 +61,7 @@
 
     /**
      * Adds a content filter based on the provided closure.  The Closure will be called with each line (stripped of line
-     * endings) and should return a String to replace the line.
+     * endings) and should return a String to replace the line or {@code null} to remove the line.
```

Comment:
Can do.  What's the preferred method for updating a PR — amend the original commit or add a new one?